package.json

```diff
@@ -27,6 +27,7 @@
   "main": "./public_api.js",
   "typings": "./public_api.d.ts",
   "dependencies": {
+    "@microsoft/api-extractor": "^6.3.0",
```

Comment:
Ideally version 7 is used. Version 6 can be challenging to use in existing projects.